src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreJob.cs

```diff
@@ -499,6 +499,9 @@ private async Task RestoreAsync(bool forceRestore, RestoreOperationSource restor
                                 {
                                     if (isRestoreSucceeded)
                                     {
+                                        var errors = restoreSummaries.SelectMany(summary => summary.Errors.Where(e => e.Code.Equals(NuGetLogCode.NU1901) || e.Code.Equals(NuGetLogCode.NU1902) || e.Code.Equals(NuGetLogCode.NU1903) || e.Code.Equals(NuGetLogCode.NU1904)));
```

Comment:
I fear this might be causing an assets file read when a project is no-oped. 
We explicitly try not to load the assets file when a project no-ops.

Reply:
I want to understand this, above we access other properties from `restoreSummaries`, (_packageCount += restoreSummaries.Select(summary => summary.InstallCount).Sum();) how is this causing a read? Is there a way to test that?

Reply:
If we're doing that, then we're probably reading it, so my concern is moot. 

I should probably look into that to see if there's a way to avoid these reads.